document_sftp/document_sftp_server.py

```diff
@@ -37,6 +38,7 @@ def check_auth_publickey(self, username, key):
                     'Ignoring key of unknown type for line %s', line)
                 continue
             if RSAKey(data=decodebytes(key_data)) == key:
+                self.env = self.env(user=user.id)
```

Comment:
note that the files which are located in the module's root (so not in `models`) are not models, but implementations of the respective interfaces that need to be implemented for paramiko